### ISSUE TITLE
website crashes when I try to install the pinterest plugin

### ISSUE THREAD

**Issue by differentlyamazinghoney** [woocommerce/pinterest-for-woocommerce#767]
### Describe the bug:
website crashes when I try to install the pinterest plugin

### Steps to reproduce:

1. Try to install pinterest pluggin for woocommerce
2. 
3. 

### Expected behavior:
Install completes and pinterest is connected to woocommerce...
### Actual behavior:
install crashes out and then your website goes down and you have to enter recovery mode and deactivate pinterest before your website comes back up

**Comment by ibndawood** [comment]
Hello @differentlyamazinghoney, Thank you for reporting this issue. It will be great if you could share your current System Status Report. You can get it by navigating to the `WooCommerce / Status` section of your site. Once there, click on the `Get system report` button and then copy it by clicking on the `Copy for support` button. Then paste it here in a comment.

**Comment by JPry** [comment]
I'm assuming this is on PHP 8+ with this error:

```
Fatal error:  Uncaught TypeError: array_key_exists(): Argument #2 ($array) must be of type array, bool given in /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/LocalFeedConfigs.php:73
Stack trace:
#0 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/LocalFeedConfigs.php(73): array_key_exists('US', false)
#1 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/LocalFeedConfigs.php(59): Automattic\WooCommerce\Pinterest\LocalFeedConfigs->initialize_local_feeds_config(Array)
#2 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/LocalFeedConfigs.php(46): Automattic\WooCommerce\Pinterest\LocalFeedConfigs->__construct()
#3 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/FeedGenerator.php(544): Automattic\WooCommerce\Pinterest\LocalFeedConfigs::get_instance()
#4 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/ProductSync.php(210): Automattic\WooCommerce\Pinterest\FeedGenerator::deregister()
#5 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/ProductSync.php(113): Automattic\WooCommerce\Pinterest\ProductSync::deregister()
#6 /Users/jpry/Sites/ventures/wp/wp-includes/class-wp-hook.php(310): Automattic\WooCommerce\Pinterest\ProductSync::maybe_deregister(Array, Array)
#7 /Users/jpry/Sites/ventures/wp/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters(NULL, Array)
#8 /Users/jpry/Sites/ventures/wp/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /Users/jpry/Sites/ventures/wp/wp-includes/option.php(570): do_action('update_option_p...', Array, Array, 'pinterest_for_w...')
#10 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/class-pinterest-for-woocommerce.php(538): update_option('pinterest_for_w...', Array)
#11 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/class-pinterest-for-woocommerce.php(522): Pinterest_For_Woocommerce::save_settings(Array)
#12 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/Admin/ActivationRedirect.php(120): Pinterest_For_Woocommerce::save_setting('did_redirect_to...', false)
#13 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/Admin/ActivationRedirect.php(68): Automattic\WooCommerce\Pinterest\Admin\ActivationRedirect->update_did_redirect_setting(false)
#14 /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/Admin/ActivationRedirect.php(40): Automattic\WooCommerce\Pinterest\Admin\ActivationRedirect->maybe_redirect_to_onboarding()
#15 /Users/jpry/Sites/ventures/wp/wp-includes/class-wp-hook.php(308): Automattic\WooCommerce\Pinterest\Admin\ActivationRedirect->Automattic\WooCommerce\Pinterest\Admin\{closure}('')
#16 /Users/jpry/Sites/ventures/wp/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters(NULL, Array)
#17 /Users/jpry/Sites/ventures/wp/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#18 /Users/jpry/Sites/ventures/wp/wp-admin/admin.php(175): do_action('admin_init')
#19 /Users/jpry/.composer/vendor/laravel/valet/server.php(110): require('/Users/jpry/Sit...')
#20 {main}
  thrown in /Users/jpry/Sites/ventures/content/plugins/pinterest-for-woocommerce/src/LocalFeedConfigs.php on line 73
```

**Comment by differentlyamazinghoney** [comment]
Should I try to reinstall the plug in and make it crash before sending the
system report? Or do you just need one in general from when its working?

**Comment by layoutd** [comment]
Hi @differentlyamazinghoney, just a general system report would be fine (as well as the version number of the plugin that's causing the issue).

**Comment by differentlyamazinghoney** [comment]
Woocommerce version 7.7.2

Heres the system report:

`
### WordPress Environment ###

WordPress address (URL): https://differentlyamazinghoney.com
Site address (URL): https://differentlyamazinghoney.com
WC Version: 7.7.2
REST API Version: ✔ 7.7.2
WC Blocks Version: ✔ 10.0.6
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.2.2
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 8.1.19
PHP Post Max Size: 516 MB
PHP Time Limit: 60
PHP Max Input Vars: 1000
cURL Version: 8.1.2
OpenSSL/1.1.1u

SUHOSIN Installed: –
MySQL Version: 5.7.23-23
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.7.2
WC Database Prefix: wpgk_
Total Database Size: 15.51MB
Database Data Size: 11.70MB
Database Index Size: 3.81MB
wpgk_woocommerce_sessions: Data: 0.09MB + Index: 0.02MB + Engine InnoDB
wpgk_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB +
Engine InnoDB
wpgk_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index:
0.06MB + Engine InnoDB
wpgk_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine
InnoDB
wpgk_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wpgk_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine
InnoDB
wpgk_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine
InnoDB
wpgk_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB +
Engine InnoDB
wpgk_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB +
Engine InnoDB
wpgk_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine
InnoDB
wpgk_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine
InnoDB
wpgk_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_actionscheduler_actions: Data: 1.52MB + Index: 0.56MB + Engine InnoDB
wpgk_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_actionscheduler_logs: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wpgk_aioseo_cache: Data: 0.13MB + Index: 0.03MB + Engine InnoDB
wpgk_aioseo_notifications: Data: 0.03MB + Index: 0.06MB + Engine InnoDB
wpgk_aioseo_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_brizy_logs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_ce4wp_abandoned_checkout: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_ce4wp_contacts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_comments: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wpgk_gla_attribute_mapping_rules: Data: 0.02MB + Index: 0.00MB + Engine
InnoDB
wpgk_gla_budget_recommendations: Data: 0.22MB + Index: 0.14MB + Engine
InnoDB
wpgk_gla_merchant_issues: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_gla_shipping_rates: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_gla_shipping_times: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_index_now_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_automations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_mailpoet_automation_runs: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_automation_run_logs: Data: 0.02MB + Index: 0.02MB + Engine
InnoDB
wpgk_mailpoet_automation_run_subjects: Data: 0.02MB + Index: 0.03MB +
Engine InnoDB
wpgk_mailpoet_automation_triggers: Data: 0.02MB + Index: 0.00MB + Engine
InnoDB
wpgk_mailpoet_automation_versions: Data: 0.02MB + Index: 0.02MB + Engine
InnoDB
wpgk_mailpoet_custom_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_dynamic_segment_filters: Data: 0.02MB + Index: 0.02MB +
Engine InnoDB
wpgk_mailpoet_feature_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_mailpoet_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_mailpoet_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_newsletters: Data: 0.19MB + Index: 0.03MB + Engine InnoDB
wpgk_mailpoet_newsletter_links: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpgk_mailpoet_newsletter_option: Data: 0.02MB + Index: 0.02MB + Engine
InnoDB
wpgk_mailpoet_newsletter_option_fields: Data: 0.02MB + Index: 0.02MB +
Engine InnoDB
wpgk_mailpoet_newsletter_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_newsletter_segment: Data: 0.02MB + Index: 0.02MB + Engine
InnoDB
wpgk_mailpoet_newsletter_templates: Data: 2.52MB + Index: 0.00MB + Engine
InnoDB
wpgk_mailpoet_scheduled_tasks: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_mailpoet_scheduled_task_subscribers: Data: 0.02MB + Index: 0.02MB +
Engine InnoDB
wpgk_mailpoet_segments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_mailpoet_sending_queues: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_mailpoet_settings: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_statistics_bounces: Data: 0.02MB + Index: 0.00MB + Engine
InnoDB
wpgk_mailpoet_statistics_clicks: Data: 0.02MB + Index: 0.05MB + Engine
InnoDB
wpgk_mailpoet_statistics_forms: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_statistics_newsletters: Data: 0.02MB + Index: 0.03MB + Engine
InnoDB
wpgk_mailpoet_statistics_opens: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wpgk_mailpoet_statistics_unsubscribes: Data: 0.02MB + Index: 0.05MB +
Engine InnoDB
wpgk_mailpoet_statistics_woocommerce_purchases: Data: 0.02MB + Index:
0.06MB + Engine InnoDB
wpgk_mailpoet_stats_notifications: Data: 0.02MB + Index: 0.03MB + Engine
InnoDB
wpgk_mailpoet_subscribers: Data: 0.02MB + Index: 0.16MB + Engine InnoDB
wpgk_mailpoet_subscriber_custom_field: Data: 0.02MB + Index: 0.02MB +
Engine InnoDB
wpgk_mailpoet_subscriber_ips: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_subscriber_segment: Data: 0.02MB + Index: 0.03MB + Engine
InnoDB
wpgk_mailpoet_subscriber_tag: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_mailpoet_tags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_user_agents: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_mailpoet_user_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_nfd_data_event_queue: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_options: Data: 4.08MB + Index: 0.14MB + Engine InnoDB
wpgk_postmeta: Data: 0.45MB + Index: 0.14MB + Engine InnoDB
wpgk_posts: Data: 0.13MB + Index: 0.06MB + Engine InnoDB
wpgk_socialsnap_stats: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_termmeta: Data: 0.06MB + Index: 0.06MB + Engine InnoDB
wpgk_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_userfeedback_surveys: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_userfeedback_survey_responses: Data: 0.02MB + Index: 0.02MB + Engine
InnoDB
wpgk_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpgk_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wpgk_wc_admin_note_actions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wpgk_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wpgk_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpgk_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine
InnoDB
wpgk_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine
InnoDB
wpgk_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wpgk_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_wd_fb_data: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_wd_fb_info: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_wd_fb_option: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_wd_fb_shortcode: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_wd_fb_theme: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wpgk_wpforms_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpgk_yoast_indexable: Data: 0.06MB + Index: 0.09MB + Engine InnoDB
wpgk_yoast_indexable_hierarchy: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpgk_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpgk_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpgk_yoast_seo_links: Data: 0.05MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

attachment: 65
brizy-project: 1
customize_changeset: 12
mailpoet_page: 1
nav_menu_item: 16
page: 11
post: 4
product: 31
revision: 31
shop_coupon: 2
wp_global_styles: 2
wp_navigation: 3
wp_template: 1
wp_template_part: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (28) ###

All in One SEO: by All in One SEO Team – 4.3.8
Breadcrumb Block: by Phi Phan – 1.0.10
Creative Mail by Newfold Digital: by Newfold Digital – 1.6.7
Customer Reviews for WooCommerce: by CusRev – 5.28.1
Facebook for WooCommerce: by Facebook – 3.0.24
Google Analytics for WordPress by MonsterInsights: by MonsterInsights – 8.15
Google Listings and Ads: by WooCommerce – 2.4.6
The Icon Block: by Nick Diego – 1.5.0
Jetpack: by Automattic – 12.2
Klarna Order Management for WooCommerce: by klarna
krokedil – 1.7.2

Klarna Payments for WooCommerce: by krokedil
klarna
automattic – 3.0.7

MailPoet: by MailPoet – 4.18.0
IndexNow: by Mikhail Kobzarev – 2.5.5
OptinMonster: by OptinMonster Popup Builder Team – 2.13.2
Really Simple SSL: by Really Simple Plugins – 7.0.3
Social Sharing Block: by Nick Diego – 0.7.0
Social Snap Lite: by Social Snap – 1.3.3
TikTok: by TikTok – 1.1.0
UserFeedback Lite: by UserFeedback Team – 1.0.5
10Web Social Feed: by 10Web – 1.2.9
WooCommerce Affirm Gateway: by WooCommerce – 2.0.2
WooCommerce Payments: by Automattic – 5.9.1
WooCommerce PayPal Payments: by WooCommerce – 2.0.5
WooCommerce Shipping & Tax: by WooCommerce – 2.2.5
WooCommerce: by Automattic – 7.7.2
Yoast SEO: by Team Yoast – 20.8
The HostGator Plugin: by HostGator – 1.2.8
WPForms Lite: by WPForms – 1.8.1.3

### Inactive Plugins (5) ###

Akismet Anti-Spam: Spam Protection: by Automattic - Anti Spam Team – 5.1
Brizy: by Brizy.io – 2.4.20
Hello Dolly: by Matt Mullenweg – 1.7.2
Klarna On-Site Messaging for WooCommerce: by krokedil
klarna – 1.11.0

Pinterest for WooCommerce: by WooCommerce – 1.3.1

### Must Use Plugins (1) ###

Endurance Page Cache: by Mike Hansen – 2.2

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### WC Pages ###

Shop base: #30 - /refund_returns/
Cart: #27 - /cart/
Checkout: #28 - /checkout/
My account: #29 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 4.3.0
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that
you did not build personally we recommend using a child theme. See: How to
create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://differentlyamazinghoney.com
Subscriptions-core Library Version: 5.7.2
Subscription Statuses: –
WooCommerce Account Connected: ❌ No

### Store Setup ###

Country / State: United States (US) — Virginia

### Subscriptions by Payment Gateway ###


### Payment Gateway Support ###

WooCommerce Payments: products
refunds
multiple_subscriptions
subscription_cancellation
subscription_payment_method_change_admin
subscription_payment_method_change_customer
subscription_payment_method_change
subscription_reactivation
subscription_suspension
subscriptions
gateway_scheduled_payments
tokenization
add_payment_method

Klarna Payments: products
refunds

PayPal: refunds

### Klarna Payments Request Log ###

No Klarna Payment errors:

### Klarna Countries ###

Production Countries: No countries selected
Test Countries: No countries selected

### WooCommerce PayPal Payments ###

Onboarded: ✔
Shop country code: US
WooCommerce currency supported: ✔
Advanced Card Processing available in country: ✔
Pay Later messaging available in country: ✔
Webhook status: –
Vault enabled: ✔
Logging enabled: –
Reference Transactions: –
Used PayPal Checkout plugin: –
Tracking enabled: ✔

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: product-block-editor
minified-js
new-product-management-experience
product-variation-management
settings

Daily Cron: ✔ Next scheduled: 2023-06-07 15:00:38 +00:00
Options: ✔
Notes: 84
Onboarding: completed

### WooCommerce Payments ###

Version: 5.9.1
Connected to WPCOM: Yes
Blog ID: 219431118
Account ID: acct_1NC4UDC7OdKhOAcg

### MailPoet ###

Sending Method: MailPoet
Send all site's emails with: Default WordPress sending method
Task Scheduler method: Action Scheduler
Cron ping URL:
https://differentlyamazinghoney.com?mailpoet_router&endpoint=cron_daemon&action=ping

### Action Scheduler ###

Complete: 1,807
Oldest: 2023-05-25 13:46:10 +0000
Newest: 2023-06-06 19:45:52 +0000

Failed: 8
Oldest: 2023-05-25 16:54:15 +0000
Newest: 2023-06-06 19:03:05 +0000

Pending: 12
Oldest: 2023-06-06 19:47:52 +0000
Newest: 2023-06-08 14:16:48 +0000


### Status report information ###

Generated at: 2023-06-06 19:45:57 +00:00
`

**Comment by layoutd** [comment]
Thanks @differentlyamazinghoney, we've released a new version (v1.3.2) that should fix this issue, and it should be available already. Please test with this new version and let us know if you still have any problems.
